run-make-check.sh

```diff
@@ -68,9 +68,6 @@ function main() {
     FOR_MAKE_CHECK=1 prepare
     # Init defaults after deps are installed.
     local cmake_opts=" -DWITH_PYTHON3=3 -DWITH_GTEST_PARALLEL=ON -DWITH_FIO=ON -DWITH_CEPHFS_SHELL=ON -DWITH_GRAFANA=ON -DWITH_SPDK=ON -DENABLE_GIT_VERSION=OFF"
-    if [ $WITH_SEASTAR ]; then
-        cmake_opts+=" -DWITH_SEASTAR=ON"
-    fi
```

Comment:
Why is this removed?

Reply:
Looks like that change is tied to this commit: https://github.com/ceph/ceph/pull/47696/commits/5b9927d271d8e6daaf7d0f72b501fe9030bde4e2

There's a description there that explains why.

Reply:
Yep, that's the one. The alternative would have been backporting crimson stuff, which @tchaikov and I thought didn't make sense for Pacific.